src/regressionmodel.jl

```diff
@@ -99,8 +121,7 @@ function effects!(reference_grid::DataFrame, model::RegressionModel;
     eff = X * coef(model)
     err = sqrt.(diag(X * vcov(model) * X'))
     if invlink !== identity
-        err .*= ForwardDiff.derivative.(invlink, eff)
-        eff .= invlink.(eff)
+        _difference_method!(eff, err, model, invlink)
     end
```

Comment:
seems like this should be an if/else since otherwise you're calculating the `err` redundantly?

Reply:
ah maybe it needs to be this way to be compatible with the other method below...